target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-whw03v2.dts

```diff
@@ -0,0 +1,555 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+    model = "Linksys WHW03 V2 (Velop)";
+    compatible = "linksys,whw03v2", "qcom,ipq4019";
+
+    aliases {
+        led-boot = &led_blue;
+        led-failsafe = &led_red;
+        led-running = &led_green;
+        led-upgrade = &led_red;
+    };
+
+    chosen {
+        bootargs-append = " root=/dev/ubiblock0_0 rootfstype=squashfs ro";
+        stdout-path = &blsp1_uart1;
+    };
+
+    memory {
+        device_type = "memory";
+        reg = <0x80000000 0x10000000>;
```

Comment:
Is the bootloader not adding the memory node?

Reply:
The memory node that was found in the DTS of the vendor firmware was different, so I have added it to this one. 
I can try to remove it and test it.

Reply:
Yeah, remove it and then you can use dtc on the running device to see the live DTS.
I would be that the bootloader will add it or update it to the correct one anyway.

Reply:
Done, it works well without it, thanks!